ming_sim/settlement_payload.py

```
@@ -87,6 +87,26 @@ def parse_decision_blocks(narrative: str) -> tuple[str, List[Dict[str, object]]]
     return clean, decisions
 
 
+def decision_has_rescript_capability(decision: object) -> bool:
+    """批红轨识别：options 同时带 dossier_id + dossier_decision（#1490/#1492 A）。
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)